client/extensions/woocommerce/state/selectors/test/promotions.js

```diff
@@ -8,12 +8,20 @@ import { expect } from 'chai';
  */
 import {
 	getPromotions,
+	getPromotionsCurrentPage,
+	getPromotionsPerPage,
 } from '../promotions';
 
 describe( 'promotions', () => {
 	const rootState = {
 		extensions: {
 			woocommerce: {
+				ui: {
+					promotions: {
+						currentPage: 12,
+						perPage: 42,
```

Comment:
Would you consider changing `42` to `30` - i.e. the usual value of perPage?

Reply:
Sure.